javascript/resources/codeql-extractor.yml

```diff
@@ -0,0 +1,94 @@
+name: "javascript"
```

Comment:
I assume this file and the scripts in the `tools` folder are copied unchanged from the internal repository and will be removed from the internal repository in the near future?

Reply:
Yes exactly, only a `CONTRIBUTING.md` file was dropped. I'm not sure what it did there in the first place tbh.